pyanaconda/payload/rpmostreepayload.py

```diff
@@ -203,6 +188,13 @@ def _install(self, data):
                     pull_opts['localcache-repos'] = Variant('as', [path])
                     break
 
+        sysroot_file = Gio.File.new_for_path(conf.target.physical_root)
```

Comment:
It is a little scary to change the order of the calls.

Reply:
Yes, it is :-/ In any case, the code in `_install` and `post_install` was the same, so I believe there is no hidden work happening, it's just constructing the objects for multiple layers of indirection. Then that must work in multiple tasks too. (I choose to be optimistic.)